### DISCUSSION TITLE
[BUG] Managed identity not working with Sites.Selected permission scope

### DISCUSSION THREAD

**Discussion by MarkoRosberg** [pnp/powershell#3663] [General]
### Reporting an Issue or Missing Feature
Managed identity not working with Sites.Selected permission scope. 

### Expected behavior 
PnP PowerShell should work with Sites.Selected scope and managed identity like with any login method.

### Actual behavior 
You'll receive "Attempted to perform an unauthorized operation." error.

### Steps to reproduce behavior
The same issue has been discussed here https://learn.microsoft.com/en-us/answers/questions/1319299/managed-identity-with-sites-selected-permission. The same issue exists with system assigned user assigned managed identity.

### What is the version of the Cmdlet module you are running?
2.2.0

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [x] Azure automation

**Comment by ToolingModus** [comment]
I can confirm that I have the same issue!

**Reply by nicole-ge** [reply]
I have the same Issue. Can someone help?
